src/components/Section.js

```diff
@@ -5,16 +5,14 @@ export default class Section {
     this._container = document.querySelector(selector);
   }
 
-  _clear() {
+  clear() {//made this public because I found a use for it. edit: No I did not. Delete works fine now. Still nice to have the nuclear option in the back of pocket, I guess?
     this._container.innerHTML = "";
   }
 
-  renderItems() {
-    //this._clear();//is this necessary?
-    /*this._items.forEach((item) => {
-      this._renderer(item);
-    });*/
-    this._items.forEach(this._renderer);//refactor of the above to be slightly more elegant.
+  renderItems() {    
+    this._items[0].forEach((item) => {//there's probably a more elegant way to do this than accessing an array position but this works well enough for me, which is to say at all. Kinda a quirk with storing the API data and needing to store/access after initializing the card section. ¯\_(ツ)_/¯
```

Comment:
`_items[0]`

**Could be improved**

Please, change `renderItems` the way it could accept the array to make it reusable in the future


```js


  renderItems(items) {
    items.forEach((item) => {
      this._renderer(item);
    });
  }

```